<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cambio_pass</class>
 <widget class="QDialog" name="cambio_pass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagen/logo.png</normaloff>imagen/logo.png</iconset>
  </property>
  <widget class="QLineEdit" name="usuario_cambio_pass">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_nuevo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Nueva contraseña</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="aceptar_cambio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagen/aceptar.png</normaloff>imagen/aceptar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_actual">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Contraseña actual</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmacion_pass_nuevo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirmación nueva contraseña</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Cambiar contraseña</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usuario_cambio_pass</tabstop>
  <tabstop>pass_actual</tabstop>
  <tabstop>pass_nuevo</tabstop>
  <tabstop>confirmacion_pass_nuevo</tabstop>
  <tabstop>aceptar_cambio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
